examples/website/plot/tsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+    "extends": "../../../tsconfig.json",
+    "include": ["./**/*.ts"],
+    "compilerOptions": { "strict": false }
```

Comment:
I opted for `strict: false` here because it makes incremental conversion easier. There were a couple tricky type issues, possibly specific to the 'examples' environment, which didn't seem urgent to deal with quite yet. For example, this line in surface-layer...

```js
return new Model(this.context.device, { ... });
```

... gives ...

```
Argument of type 'import("/Users/donmccurdy/Projects/deck.gl/node_modules/@luma.gl/core/dist/adapter/device").Device' is not assignable to parameter of type 'import("/Users/donmccurdy/Projects/deck.gl/examples/website/plot/node_modules/@luma.gl/core/dist/adapter/device").Device'.
  Property 'getCanvasContext' is missing in type 'import("/Users/donmccurdy/Projects/deck.gl/node_modules/@luma.gl/core/dist/adapter/device").Device' but required in type 'import("/Users/donmccurdy/Projects/deck.gl/examples/website/plot/node_modules/@luma.gl/core/dist/adapter/device").Device'.ts(2345)
device.d.ts(133, 5): 'getCanvasContext' is declared here.
```

With this combination of `strict: false` and some `as $TODO`, the type checker is happy with the PR.

Reply:
Try and resolve the remaining issues so we don't need this. I assume you're running the example using `yarn start` in the example directory. Also test running the website (`cd deck.gl/website && yarn && yarn start`). It is currently failing